website/docs/examples/track_planner/README.md

```diff
@@ -0,0 +1,107 @@
+---
+id: track-planner
+title: Track Planner
+---
+
+:::info Basic Knowledge Requirements
+
+Before diving into this code, here's a quick heads-up on what you'll need to be familiar with:
+
+1. **Python Programming**: It's important to have a good grasp of Python, especially with concepts
+like `functions`, `loops`, and `classes`, since the example utilizes these fundamentals.
+2. **Asynchronous Programming with asyncio**: Familiarity with Python's asyncio for writing concurrent
+code using the `async/await` syntax.
+3. **[farm-ng Filter Service Overview](/docs/concepts/filter_service/)**:
+This overview provides a base understanding of the gRPC service the client you create will connect to.
+4. [**farm-ng Transforms & Poses Overview**](/docs/concepts/transforms_and_poses/):
+This overview provides insight into coordinate frames, transforms,
+and poses as they pertain to autonomous systems and autonomous navigation.
+5. [**farm-ng Tracks & Waypoints Overview**](/docs/concepts/tracks_and_waypoints/):
+This overview provides insight into compiling poses as waypoints into a Track
+that can be followed by the Amiga.
+:::
+
+The [**Track Planner Example**](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/track_planner/main.py)
+operates as a standalone Python script,
+in which a [**`Track`**](https://github.com/farm-ng/farm-ng-amiga/blob/main/protos/farm_ng/track/track.proto)
+proto message is generated using the [TrackPlanner](https://github.com/farm-ng/farm-ng-amiga/blob/main/py/examples/track_planner/track_planner.py)
+class. We use the [`matplotlib`](https://matplotlib.org/) library to visualize
+the Track we created.
+
+We used this example to generate a virtual strawberry field at the 2024 World Ag. Expo!
+
+To successfully run this example, you must use your local PC, as the example won't
+work if executed directly from a brain (because of the popup window).
+
+## 1. Install the [farm-ng Brain ADK package](/docs/brain/brain-install)
+
+## 2. Install the example's dependencies
+
+:::tip
+It is recommended to also install these dependencies and run the
+example in the brain ADK virtual environment.
+:::
+
+### Setup
+
+:::important Recommended
+Create a virtual environment
+:::
+
+```bash
+python3 -m venv venv
+source venv/bin/activate
+```
+
+### Install
+
+```bash
+cd py/examples/track_plotter
+pip install -r requirements.txt
+```
+
+## 3. Execute the Python script
+
+```bash
+python main.py --service_config service_config.json
+```
+
+:::info
+Since this example must be run from your local PC, you will need update the `service_config.json`
+by modifying the `host` field with your Amiga brain name.
+
+Please check out [Amiga Development 101](docs/concepts/system_overview/README.md#running-examples-on-your-local-machine)
+for more details.
```

Comment:
service_config.json should be service-config.json otherwise it doesn't work.  Add instructions for installing a module named 'tkinter' as it is one of the required dependencies